tests/test_client.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
```

Comment:
Would be cool if someone would work with me on integration tests. Those test should be able to stop and start single nodes of a cluster in order to simulate failures. Currently I have no idea on how to do this